scripts/updateConsent.js

```diff
@@ -0,0 +1,29 @@
+var fs = require('fs');
+const {Pool} = require('pg');
+
+function updateConsent(database){
+  //Still need to grab emails from a sensitive data file
+  const emails = JSON.parse(fs.readFileSync('./tmp/consented-latest.json'))
+  const emailString = emails.consented.map(x => "'" + x + "'").toString();
```

Comment:
⚠️ Anytime you're making SQL with strings, you're allowing SQL injection. ⚠️ 

In general, escaping is really hard to do right, and you should always use built-in functions to do this in SQL or any other language.  Check out https://stackoverflow.com/a/2049510 to see all of the craziness involved in something like email addresses where it might seem relatively safe to just concatenate, but there are a million edge cases that will work around just adding `'` around the value.  Escaping strings yourself is really hard to do correctly and securely.

I don't think `unnest` isn't doing what you want it to do here anyway.  It's taking a single string value `'kevin@mit.edu', 'tim@mit.edu'` and inserting that as the `email` column for a single row, like this:

```
 id |             email             | audio | permission | consent
----+-------------------------------+-------+------------+---------
  1 | 'foo@bar.com','kevin@mit.edu' | t     | t          | t
```

Googling "node postgres insert multiple values" leads to some suggestions here: https://github.com/brianc/node-postgres/issues/957.  You could start with mapping over the values to running a separate query for each, and then return a `Promise.all` to wait for them.